### PR TITLE
Update provider privacy policy content

### PR DESCRIPTION
**Pull request by SamJamCul**
## Context

[](https://trello.com/c/VcH5LfAE/4558-using-the-right-privacy-policy)

The content that I used for the previous [privacy policy PR](https://github.com/DFE-Digital/apply-for-teacher-training/pull/6087) was out of date! This now brings it in line with the most up to date version of the privacy policies. 

## Changes proposed in this pull request

Updating content for /provider/privacy policies

## Guidance to review
Add /provider/privacy to the url, and check both policies are up to date!

### CODE REVIEW

File: app/views/content/online_chat_privacy_notice.md [hunk 1]
```diff
@@ -42,39 +44,37 @@ We’ll also analyse and consolidate data from support requests to help us under
...
 
-Under the DPA 2018 and UK GDPR and the lawful basis of public interest, you have the right to:
+_ find out what data we hold about you
+_ be informed about how your data is being used
+_ access personal data
+_ have incorrect or incomplete data updated
+_ object to how your data is processed
+_ restrict the processing of your data in certain circumstances
+_ object to direct marketing
+_ object to automated decision making and profiling
```
Comment (despo) [inline app/views/content/online_chat_privacy_notice.md:64]:
```suggestion
- find out what data we hold about you
- be informed about how your data is being used
- access personal data
- have incorrect or incomplete data updated
- object to how your data is processed
- restrict the processing of your data in certain circumstances
- object to direct marketing
- object to automated decision making and profiling
```

Comment (SamJamCul) [reply app/views/content/online_chat_privacy_notice.md]: Thanks Despo, I was having trouble deciding between `-` and `*` and ended up with `_`!

**Comment by frankieroberto** [comment]
@SamJamCul given that this applies to candidates too, can we merge the file back into `privacy_policy.md` and use the same content for both?

**Comment by SamJamCul** [comment]
> @SamJamCul given that this applies to candidates too, can we merge the file back into `privacy_policy.md` and use the same content for both?

Sure thing - I'll move back to using the same file, which will be used in both cases

**Review by cwrw — APPROVED** [review]
👍  everyone happy :D?

**Pull request merged**